appengine/genetics.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="google" value="notranslate">
+  <meta name="viewport" content="target-densitydpi=device-dpi, width=device-width, initial-scale=1.0, user-scalable=no">
+  <title>Blockly Games : Genetics Lab</title>
+  <link rel="stylesheet" href="common/common.css">
+  <link rel="stylesheet" href="genetics/style.css">
+  <script src="third-party/ace/ace.js"></script>
+  <script src="third-party/ace/ext-language_tools.js"></script>
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
+  <script src="common/boot.js"></script>
+  <!--Leading slash is needed so that if opened locally as opposed to appengine it will not 404.-->
```

Comment:
Changed to "Leading slash so that if not loaded in appengine it will 404 instead of opening locally"